ivy/container/random.py

```diff
@@ -58,3 +58,57 @@ def random_uniform(
             map_sequences,
             out=out,
         )
+
+
+    # randint
+    @staticmethod
+    def static_randint(
+        low: Union[int, ivy.Container] = 0.0,
+        high: Union[int, ivy.Container] = 1.0,
+        shape: Optional[Union[int, Tuple[int, ...], ivy.Container]] = None,
+        device: Optional[Union[ivy.Device, ivy.NativeDevice, ivy.Container]] = None,
+        key_chains: Optional[Union[List[str], Dict[str, str]]] = None,
+        out: Optional[ivy.Container] = None,
+    ) -> ivy.Container:
+        """
+        ivy.Container static method variant of ivy.randint. This method simply wraps the
+        function, and so the docstring for ivy.randint also applies to this method
+        with minimal changes.
+
+        Examples
+        --------
+        With no device argument and no out argument specified and int arguemnt for shape:
+
+        >>> x = ivy.randint(low=1, high=10, shape=2)
```

Comment:
In examples for `static_randint`, the examples should be of the form `ivy.Container.static_randint`, where `low` and `high` passed to it are also `ivy.Container`, something of this sort,
`ivy.Container.static_randint(low=ivy.Container(a=2, b=4), high=ivy.Container(a=3, b=6), shape=(3,2))`
Please update the examples you've added to bring them to a similar form to this, basically pass `ivy.Container` to `low` and `high` parameters of the `ivy.Container.static_randint` method.

Rest looks good to me! Let me know if you have any questions. Thanks 🙂